src/hooks/useScrollToTop.jsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+
+const useScrollToTop = () => {
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
+  }, []);
```

Comment:
suggestion - here you can give a dependency of pathname such that it does not run once the component gets rendered. you can reproduce this issue by going to questions page, scroll to bottom and then click on next question